test/caching-precompiler.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 const fs = require('fs');
 const path = require('path');
-const test = require('tap').test;
+const tap = require('tap');
```

Comment:
Could you revert this change?

Reply:
oh I thought i reverted this, sorry about that, yeah.